src/cmd-build-fast

```diff
@@ -90,16 +90,22 @@ echo "Basing on previous build: ${previous_build:-none}"
 
 if [ -n "${projectdir}" ]; then
     cd "${projectdir}"
-    rm _install -rf
+    instroot=_install
+    # Rust projects will already have a handy directory for build artifacts
+    if test -d target; then
```

Comment:
Optional:

```suggestion
    if test -f Cargo.toml && test -d target; then
```

Reply:
Makes sense, but will roll this into a followup to avoid a whole CI cycle just for this